h2push.php

```diff
@@ -51,7 +51,7 @@ function start_output_buffer(): void {
  */
 function stop_output_buffer(): void {
 	if ( ob_get_length() ) {
-		ob_flush();
+		echo ob_get_clean(); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Wieso verwenden wir `ob_end_flush()` nicht?
```suggestion
		ob_end_flush();
```